nipy/algorithms/clustering/imm.py

```diff
@@ -234,7 +236,7 @@ def cross_validated_update(self, x, z, plike, kfold=10):
         if np.isscalar(kfold):
             aux = np.argsort(np.random.rand(n_samples))
             idx = - np.ones(n_samples).astype(np.int)
-            j = np.ceil(n_samples / kfold)
+            j = int(math.ceil(n_samples / kfold))
```

Comment:
Yes, will be an integer in Python 2.   I guess the question for @bthirion is - was this intended to be true (floating point division) even in Python 2?   At the moment this can give different answers on Python 2 and Python 3.